src/lib/components/LoginModal.svelte

```diff
@@ -32,20 +35,27 @@
 			AI is an area of active research with known problems such as biased generation and
 			misinformation. Do not use this application for high-stakes decisions or advice.
 		</p>
-		<p class="px-2 text-sm text-gray-500">
-			Your conversations will be shared with model authors unless you disable it from your settings.
-		</p>
+		{#if PUBLIC_APP_DATA_SHARING}
+			<p class="px-2 text-sm text-gray-500">
+				Your conversations will be shared with model authors unless you disable it from your
+				settings.
+			</p>
+		{/if}
 		<form
 			action="{base}/{$page.data.requiresLogin ? 'login' : 'settings'}"
 			target={isIframe ? "_blank" : ""}
 			method="POST"
+			class="flex w-full flex-col items-center gap-2"
 		>
 			{#if $page.data.requiresLogin}
 				<button
 					type="submit"
-					class="mt-2 flex items-center whitespace-nowrap rounded-full bg-black px-5 py-2 text-lg font-semibold text-gray-100 transition-colors hover:bg-yellow-500"
+					class="mt-2 flex items-center whitespace-nowrap rounded-full bg-black px-5 py-2 text-lg font-semibold text-gray-100 transition-colors hover:bg-primary-500"
 				>
-					Sign in with <LogoHuggingFaceBorderless classNames="text-xl mr-1 ml-1.5" /> Hugging Face
+					Sign in
+					{#if PUBLIC_APP_ASSETS == "huggingchat"}
```

Comment:
I think I'd rather use `PUBLIC_APP_NAME` than `PUBLIC_APP_ASSETS` for this.

Reply:
ok, done!